content/release-10-14-0/announcements-10-14-0.md

```diff
@@ -6,6 +6,15 @@ layout: bundle
 
 ### REST API changes
 
+#### Planned
+
+##### Breaking change in the Application API - blocking ability to change application key and contextPath
```

Comment:
```suggestion
##### Breaking change in the Application API - blocking the ability to change application key and contextPath
```
i think adding the article makes it a bit clearer. however, it's still not apparent from the heading who is blocking the ability. maybe we could add "we are ..." after the dash?